codelists/models.py

```diff
@@ -313,6 +313,13 @@ def get_diff_url(self, other_clv):
         kwargs["other_tag_or_hash"] = other_clv.tag_or_hash
         return reverse(f"codelists:{self.codelist_type}_version_diff", kwargs=kwargs)
 
+    def exists(self):
+        """
+        Check that a CodelistVersion exists.  It may not, in the event that it's just
```

Comment:
Might be clearer if this said "Check that this CodelistVersion exists in the database".